exchanges/okx/okx_test.go

```diff
@@ -0,0 +1,3129 @@
+		t.SkipNow()
+	}
+	if _, err := ok.CancelAllRFQs(context.Background()); err != nil &&
+		!strings.Contains(err.Error(), "code: 70006 message: Does not meet the minimum asset requirement.") {
+		t.Errorf("%s CancelAllRFQs() error %v", ok.Name, err)
+	}
+}
+
+func TestExecuteQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.SkipNow()
+	}
+	_, err := ok.ExecuteQuote(context.Background(), ExecuteQuoteParams{})
+	if err != nil && !errors.Is(err, errMissingRfqIDOrQuoteID) {
+		t.Errorf("Okx ExecuteQuote() expected %v, but found %v", errMissingRfqIDOrQuoteID, err)
+	}
+	if _, err = ok.ExecuteQuote(context.Background(), ExecuteQuoteParams{
+		RfqID:   "22540",
+		QuoteID: "84073",
+	}); err != nil && !strings.Contains(err.Error(), "code: 70006 message: Does not meet the minimum asset requirement.") {
+		t.Error("Okx ExecuteQuote() error", err)
+	}
+}
+
+func TestSetQuoteProducts(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.SetQuoteProducts(context.Background(), []SetQuoteProductParam{
+		{
+			InstrumentType: "SWAP",
+			Data: []MakerInstrumentSetting{
+				{
+					Underlying:     "BTC-USD",
+					MaxBlockSize:   10000,
+					MakerPriceBand: 5,
+				},
+				{
+					Underlying: "ETH-USDT",
+				},
+			},
+		}}); err != nil && !strings.Contains(err.Error(), "code: 70006 message: Does not meet the minimum asset requirement.") {
+		t.Errorf("%s SetQuoteProducts() error %v", ok.Name, err)
+	}
+}
+
+func TestResetMMPStatus(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.ResetMMPStatus(context.Background()); err != nil && !strings.Contains(err.Error(), "No permission to use this API") {
+		t.Errorf("%s ResetMMPStatus() error %v", ok.Name, err)
+	}
+}
+
+func TestCreateQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.CreateQuote(context.Background(), CreateQuoteParams{}); err != nil && !errors.Is(err, errMissingRfqID) {
+		t.Errorf("Okx CreateQuote() expecting %v, but found %v", errMissingRfqID, err)
+	}
+	if _, err := ok.CreateQuote(context.Background(), CreateQuoteParams{
+		RfqID:     "12345",
+		QuoteSide: order.Buy,
+		Legs: []QuoteLeg{
+			{
+				Price:          1234,
+				SizeOfQuoteLeg: 2,
+				InstrumentID:   "SOL-USD-220909",
+				Side:           order.Sell,
+			},
+			{
+				Price:          1234,
+				SizeOfQuoteLeg: 1,
+				InstrumentID:   "SOL-USD-220909",
+				Side:           order.Buy,
+			},
+		},
+	}); err != nil && !strings.Contains(err.Error(), "code: 70006 message: Does not meet the minimum asset requirement.") {
+		t.Errorf("%s CreateQuote() error %v", ok.Name, err)
+	}
+}
+
+func TestCancelQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	if _, err := ok.CancelQuote(context.Background(), CancelQuoteRequestParams{}); err != nil && !errors.Is(err, errMissingQuoteIDOrClientSuppliedQuoteID) {
```

Comment:
Please add a valid scenario.
Please protect this with `canManipulateRealOrders`